sdk/cosmosdb/cosmos/src/client/SasToken/SasTokenProperties.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { SasTokenPartitionKeyValueRange } from "./SasTokenPartitionKeyValueRange";
+import { CosmosContainerChildResourceKind } from "../../common/constants";
+import { CosmosKeyType } from "../../common/constants";
+
+export interface SasTokenProperties {
```

Comment:
Remove this file. In TS is it is not common to have both an interface and class if there is only a single implementation. The class already defines this interface so this file is not needed.